configs/example_configs_validation.py

```diff
@@ -0,0 +1,50 @@
+import os
+import pathlib
+import subprocess
+import sys
+import traceback
+
+import yaml
+
+from google.protobuf.json_format import ParseError
+
+sys.path = [p for p in sys.path if not p.endswith('bazel_tools')]
+
+from tools.config_validation.validate_fragment import ValidateFragment
+
+
+def main():
+  parse_errors = []
+  other_errors = []
+  for arg in sys.argv[1:]:
+    if not arg.endswith(".yaml"):
+      continue
+    try:
+      ValidateFragment("envoy.config.bootstrap.v3.Bootstrap",
+                       yaml.safe_load(pathlib.Path(arg).read_text()))
+    except ParseError as e:
+      parse_errors.append(arg)
+      print()
+      print(f"ERROR (parse failed): {arg}")
+      print(e)
+      print()
+    except Exception as e:
```

Comment:
this seems to be necessary as sometimes upstream protobuf validation throws primitive errors - eg `KeyError` - as this is a bit indeterminate i dont see any other way to catch than the  antipattern of `except Exception`

Reply:
hmm - actually going to change this to just catch keyerror - other errors should raise